src/pages/PoolsPage/index.tsx

```diff
@@ -1,7 +1,9 @@
+import PoolTable from "./PoolTable";
+
 const PoolsPage = () => {
   return (
-    <div className="flex justify-center">
-      <h1>Pool list</h1>
+    <div className="w-full px-28">
```

Comment:
We should use here width logic rather than use it in the table component.

Reply:
done